odxtools/parameters/reservedparameter.py

```diff
@@ -32,11 +34,11 @@ def bit_length(self) -> int:
         # need to take the "bit_length_raw" detour...
         return self.bit_length_raw
 
-    def get_coded_value_as_bytes(self, encode_state):
+    def get_coded_value_as_bytes(self, encode_state: EncodeState) -> bytes:
         bit_position_int = self.bit_position if self.bit_position is not None else 0
         return (0).to_bytes((self.bit_length + bit_position_int + 7) // 8, "big")
 
-    def decode_from_pdu(self, decode_state: DecodeState):
+    def decode_from_pdu(self, decode_state: DecodeState) -> Tuple[Any, int]:
```

Comment:
"Any" ?

Reply:
Should be `ParameterValue`. I wonder why mypy does not complain. fixed: 83d105ce0e80d59618dec29b5002e9fc6f75fb89